docs/release-notes/next-hc/RELEASE-NOTES-0.9.9.md

```diff
@@ -0,0 +1,90 @@
+# Hyperchains
+
+This is release of Hyperchains as described in [Periodically Syncing HyperChains](https://github.com/aeternity/hyperchains-whitepaper/blob/master/Periodically-Syncing-HyperChains.md).
+See also [aeternity Hyperchains](https://aeternity.com/hyperchains).
+
+Hyperchains integrate with other blockchain networks, leveraging their security and consensus mechanisms while providing scalable, energy-efficient hyperchains.
+
+An aeternity Hyperchain comes with all the normal features of aetenity:
+* FATE VM: Efficient virtual machine for executing smart contracts.
+* Sophia Language: A functional programming language for secure smart contracts.
+* State Channels: Off-chain scaling for private and secure transactions.
+* AENS: Æternity Naming System for human-readable addresses.
+* Oracles: Bridging real-world data into the blockchain ecosystem.
+
+See [aeternity.com](aeternity.com) for more information.
+
+# About this release
+
+This preliminary implementation of Hyperchains lays the foundation for a scalable and energy-efficient blockchain system. Key features introduced in this release include:
+
+## Core Features
+1. **Staking Mechanism**
+   - Enables participants to stake tokens and qualify for block production and validation. The staking framework ensures fairness and rewards based on contributions and performance.
+
+2. **Periodic Synchronization with Pinning Mechanism**
+   - Introduces a semi-lockstep synchronization model that aligns the epochs of Hyperchains with the pinning chain.
+   - A cryptographically secure **pinning mechanism** anchors the hyperchain state to the pinning chain, ensuring robust alignment and reduced operational overhead.
+
+3. **Future Leader Election**
+   - Implements a randomized leader election system, utilizing entropy derived from the pinning chain.
+   - This ensures predictable and secure block production, enhancing efficiency and reducing the need for frequent consensus actions.
+
+4. **Finality Guarantees**
+   - **Fast Finality:** When all validators are online and behaving correctly, blocks achieve instant finality.
+   - **Detectable Finality:** Validators can confirm block finality immediately after production.
+   - **Semi-Fast Finality:** In scenarios with offline or malicious validators, finality is limited to one epoch, ensuring operational continuity.
+
+5. **High Throughput**
+   - Optimized system design supports processing a high volume of transactions, meeting the demands of modern blockchain applications.
+
+## Governance and Consensus Enhancements
+1. **On-Chain Governance**
+   - Validators with locked stakes can vote on crucial parameters like fork resolution and epoch length adjustments, promoting decentralized decision-making.
```

Comment:
Nitpicking, not sure fork resolution is a parameter. And I don't think we'll be able to finish epoch length adjustments in the first version.